vllm/utils.py

```
@@ -109,8 +111,9 @@ def touch(self, key: Hashable) -> None:
     def get(self,
             key: Hashable,
             default_value: Optional[T] = None) -> Optional[T]:
+        value: Optional[T]
```

this doesn't seem to be optional anymore this way?

---

(iiuc if there's no item, it should raise an exception?)

---

Yes, that is the intention so it works more like a Python dictionary.